source/grid/grid_generator.inst.in

```diff
@@ -69,6 +69,15 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
         Triangulation<deal_II_dimension, deal_II_space_dimension> &,
         const std::vector<unsigned int> &);
 
+      template void
+      merge_triangulations(
+        const std::initializer_list<
+          const Triangulation<deal_II_dimension, deal_II_space_dimension>
+            *const> triangulations,
+        Triangulation<deal_II_dimension, deal_II_space_dimension> &result,
+        const double duplicated_vertex_tolerance = 1.0e-12,
+        const bool   copy_manifold_ids           = false);
```

Comment:
I guess the default values and object names should be removed from the instantiation line.